lib/simp/beaker_helpers.rb

```diff
@@ -240,15 +240,15 @@ def pfact_on(sut, fact_name)
     else
       facts_json = nil
       begin
-        cmd_output = on(sut, 'facter -p --json', :silent => true)
+        cmd_output = retry_on(sut, 'facter -p --json', :silent => true)
```

Comment:
This is going to retry 60 times with a retry interval of 1 second.  Perhaps we should check the output to see if 'no longer supported' message exist before retrying.  Could check that in a block passed to retry_on?

Reply:
Hmm...I'll check

Reply:
Doesn't look like I can pass a block but I can certainly lower the retries and interval.

Reply:
I switched it back. Doesn't make sense to try more than once in this case honestly.